packages/quality-gates/tests/test_readme.py

```
@@ -0,0 +1,71 @@
+    assert json_blocks, "README に ```json コードブロックが見つかりません"
+
+    for block in json_blocks:
+        json.loads(block)  # 無効な JSON なら json.JSONDecodeError で失敗する
+
+
+def test_readme_clarifies_threshold_scope() -> None:
+    """test_file_threshold / test_line_threshold が test-gate-checker.py 専用で
+    post-implementation-review.py には適用されないことを明記していることを確認する。
+
+    Issue #134 レビュー指摘: 旧文言はレビュー提案にも適用されるかのように
+    読めたが、post-implementation-review.py は固定定数（FILE_THRESHOLD /
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)